internal/resource/http.go

```diff
@@ -117,6 +120,12 @@ func (f *Fetcher) UpdateHttpTimeoutsAndCAs(timeouts types.Timeouts, cas []types.
 
 	f.client.transport.TLSClientConfig = &tls.Config{RootCAs: pool}
 	f.client.client.Transport = f.client.transport
```

Comment:
Does this need to be dropped as well?

Reply:
You were right, took it out and tests pass. Change addressed in latest rebase.